src/coreclr/src/utilcode/clrhost_nodependencies.cpp

```diff
@@ -69,26 +69,19 @@ thread_local ClrDebugState* t_pClrDebugState;
 // Fls callback to deallocate ClrDebugState when our FLS block goes away.
 void FreeClrDebugState(LPVOID pTlsData)
 {
-#ifdef _DEBUG
```

Comment:
Why do we turn this on for Release builds now?

Reply:
This ifdef was redundant. This whole function is debug-only (ifdef a few lines above).